drivers/overlay/overlay.go

```diff
@@ -292,6 +292,15 @@ func Init(home string, options graphdriver.Options) (graphdriver.Driver, error)
 		backingFs = fsName
 	}
 
+	runhome := filepath.Join(options.RunRoot, filepath.Base(home))
+	supported, err := SupportsNativeOverlay(home, runhome)
+
+	if !supported {
+		if path, err := exec.LookPath("fuse-overlayfs"); err == nil && opts.mountProgram == "" {
+			opts.mountProgram = path
```

Comment:
if `mountProgram` is already configured, we don't have to override it.

Reply:
Indeed, the user might have specified a different value and we don't want to remove that one.